Dockerfile-QT

```diff
@@ -5,15 +5,15 @@ WORKDIR ${WORK_DIR}
 
 COPY . ${WORK_DIR}/
 
-RUN mkdir /opt/jboss/keycloak/themes/cwa
-RUN mkdir /opt/jboss/keycloak/themes/quick-test
-RUN cp -r /opt/jboss/keycloak/themes/base/* /opt/jboss/keycloak/themes/cwa/
-RUN cp -r ${WORK_DIR}/src/themes/cwa/login /opt/jboss/keycloak/themes/cwa/
-RUN cp -r ${WORK_DIR}/src/themes/cwa/account /opt/jboss/keycloak/themes/cwa/
-RUN cp -r /opt/jboss/keycloak/themes/base/* /opt/jboss/keycloak/themes/quick-test/
-RUN cp -r ${WORK_DIR}/src/themes/quick-test/login /opt/jboss/keycloak/themes/quick-test/
-RUN cp -r ${WORK_DIR}/src/themes/quick-test/account /opt/jboss/keycloak/themes/quick-test/
-RUN cp ${WORK_DIR}/src/standalone/configuration/standalone-ha-qt.xml /opt/jboss/keycloak/standalone/configuration/standalone-ha.xml
```

Comment:
The `-qt` postfix is deleted here which makes the `standalone-ha-qt.xml` configuration file redundant. I just wanted to flag in case it was a typo.

_Note: I have no information about the project, I was just checking various Keycloak implementations on GitHub._

Reply:
Thanks a lot!!! Yes this is an error. I've created a PR to fix this.

https://github.com/corona-warn-app/cwa-verification-iam/pull/76